.travis.yml

```diff
@@ -24,6 +24,10 @@ env:
   global:
     - MIX_ENV: test
     - DATABASE_URL: postgres://localhost/elixir_boilerplate_test
+    - PORT: 4000
+    - SECRET_KEY_BASE: 6Z6Tv8zu0vg+D9I7WU6e7sl0HMqvq1lXDDodLTLSZogGGfwPSoNJHCyipz7XVNbi
+    - SESSION_KEY: elixir_boilerplate
+    - SIGNING_SALT: /gdYXAjk3FYyLWrL+CQCzuTEmmyouMYZp1BzYq2RNZQo4bKb1yelp5MwLzq3Zd5I
```

Comment:
Why were these variables moved from `config/test.exs` to here? They were there for a very good reason 😄

Reply:
The config lifecycle changed with Mix releases. The `config/releases.exs` script is automatically invoked, following the `config/MIX_ENV.exs` of the environment in the release bootstrap.

Which means I had to remove the import from `config/config.exs` otherwise the script was executed twice.

https://github.com/mirego/elixir-boilerplate/pull/67/files#diff-de9210b77e43a621a256d7a37fc5f50aL24-L26

I moved it to environment we are not running as relases (_ie_ `:dev`, `:test`)

https://github.com/mirego/elixir-boilerplate/pull/67/files#diff-3c915f1e59382302a6c4b73bc03e9ee7R20-R22

Which caused the hard coded values from `config/test.exs` to be overwritten by the import…

Reply:
Could we just move the 

```elixir
# Import runtime configuration
import_config "releases.exs"
```

block at the _top_ of `config/dev.exs` and `config/test.exs` instead of at the _end_ of them?

Reply:
Of course, I’ll give it a try!